.github/workflows/deploy_edgenet.yaml

```diff
@@ -22,8 +23,8 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.EDGENET_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.EDGENET_AWS_SECRET_ACCESS_KEY }}
         run: >
-          /usr/local/bin/aws-commander 
-          -instances i-039f7c0b3328a00f8,i-035b9f2d78cfb8ea9,i-00a6c7cb3a213f21f,i-03ac2f42ddcba6120 
-          -mode ansible 
-          -playbook .github/workflows/ansible/update_edgenet.yaml 
+          /usr/local/bin/aws-commander
+          -instances i-039f7c0b3328a00f8,i-035b9f2d78cfb8ea9,i-00a6c7cb3a213f21f,i-03ac2f42ddcba6120

```

Comment:
Could we implement a dynamic inventory based on tags here?

Reply:
Yes we can. Edgenet deployment wasn't in this PR scope but we discussed about using tags or resource groups initially.

I can implement this logic for the `VALIDATOR_ASG` secret variable though for the proposed `devnet` and `testnet` deployemnts. :pray: Thanks for the suggestion!